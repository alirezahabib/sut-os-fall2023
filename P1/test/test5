/usr/bin/sleep 2 &
/usr/bin/ps -o cmd
wait
echo hey
