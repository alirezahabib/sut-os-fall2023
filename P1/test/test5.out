CMD
python3.9 tester.py
./shell
/usr/bin/sleep 2
/usr/bin/ps -o cmd
hey
